lib/smart_proxy_abrt/abrt_lib.rb

```diff
@@ -2,13 +2,17 @@
 require 'net/https'
 require 'uri'
 require 'time'
+require 'yaml'
 require 'fileutils'
 
 require 'openssl'
 
 require 'proxy/log'
 require 'proxy/request'
 
+$settings = YAML::load(File.open('/etc/foreman-proxy/settings.d/abrt.yml'))
```

Comment:
I'd leave the actual module as is, esp. considering that when latest module initialization is used, settings are passed  in as constructor parameter.

Reply:
tl;dr the module doesnt work right now